.github/workflows/assemble.yml

```diff
@@ -30,7 +30,5 @@ jobs:
 
       - name: Build All Projects
         run: |
-          bundle install
-          fastlane build_subscriber
-          fastlane build_publisher
-          fastlane build_example_apps
+            bundle install
+            ./Scripts/./assemble.sh
```

Comment:
I think you could remove the `./` in the middle :wink: So it would look like this `./Scripts/assemble.sh`

Reply:
here 7605cbe9eed6324568ac98188ee2faa9ea2c2c70